script/Arcueid/Day 10/10_00_ARC10_1.txt

```diff
@@ -99,7 +99,7 @@ My mind is so blank it's as though my brain has turned into tofu.
 [sha:932c1ea514240741390bb70b22bbde61b4fe0fc5]{
 -- Page 6, Offset 24278.
 -- 口は呼吸さえ忘れて固まっている。
-My mouth forgets how to even breathe and seizes up.
+Even my mouth forgets how to breathe and seizes up.
```

Comment:
Prefer original, this form reads a bit more like there's other things that are also forgetting how to breathe